src/amo/components/CollectionManager/index.js

```diff
@@ -409,7 +409,6 @@ export class CollectionManagerBase extends React.Component<
             inputPlaceholder={i18n.gettext(
               'Find an add-on to include in this collection',
             )}
-            location={router.location}
```

Comment:
Do we need the `router` somewhere? If not, we can probably remove the `withRouter` HOC.

Reply:
We use `router` in `onCancel`.